deeplabcut/create_project/modelzoo.py

```diff
@@ -261,7 +261,9 @@ def create_pretrained_project(
         download_huggingface_model(model, train_dir)
 
         pose_cfg = deeplabcut.auxiliaryfunctions.read_plainconfig(path_train_config)
-        pose_cfg["dataset_type"] = "imgaug"
+        # Add some multi-animal defaults
+        pose_cfg["nmsradius"] = 5.0
+        pose_cfg["sigma"] = 1
```

Comment:
Revert this change: it's not needed with the line above pose_cfg["dataset_type"] = "imgaug"